dask/dataframe/tests/test_dataframe.py

```diff
@@ -1335,6 +1335,39 @@ def test_deterministic_apply_concat_apply_names():
             sorted(aca(a.x, f, f, a.x.name).dask))
 
 
+def test_dataframe_apply_concat_apply():
+    df_pd = pd.DataFrame({'v1': [1, 3, 5, 7], 'v2': [987, 78, 13, 99]})
+    df = dd.from_pandas(df_pd, npartitions=2)
+    minimum = df.apply_concat_apply(lambda x: x.min()).compute()
+    expected = pd.Series({'v1': 1, 'v2': 13})
+    eq(minimum, expected)
+
+    min_max = df.apply_concat_apply(
+        lambda x: x.min(),  aggfunc=lambda x: x.max()).compute()
+    expected = pd.Series({'v1': 5, 'v2': 78})
+    eq(min_max, expected)
+
+
+def test_series_apply_concat_apply():
+    df_pd = pd.DataFrame({'val': range(500)})
+    df = dd.from_pandas(df_pd, npartitions=4)
+    count = df['val'].apply_concat_apply(lambda x: x.count(),
+                                         aggfunc=lambda x: x.sum()).compute()
```

Comment:
It might be wise to look at or merge that PR before reviewing these comments in depth.